ecs-init/config/common.go

```diff
@@ -155,3 +155,11 @@ func HostCertsDirPath() string {
 	}
 	return hostCertsDirPath
 }
+
+// HostPKIDirPath() returns the CA store path on the host
+func HostPKIDirPath() string {
+	if _, err := os.Stat(hostPKIDirPath); err != nil {
```

Comment:
It looks like `hostPKIDirPath` is missing from `ecs-init/config/config_suse_ubuntu.go`, so compilation using those tags will fail.  Can you add a definition and then check here for `hostPKIDirPath != ""` as well?

Reply:
sigh, fixing.